experimental/coref/test_project_coref.py

```diff
@@ -0,0 +1,22 @@
+import os
+from spacy.cli.project.run import project_run
+from spacy.cli.project.assets import project_assets
+from pathlib import Path
+
+
+def test_experimental_coref():
+
+    # This is set globally in testing to make sure tests are fast, but can't be
+    # used for this project because passing training-specific parameters breaks
+    # the spacy assemble command.
+    os.environ["SPACY_CONFIG_OVERRIDES"] = ""
```

Comment:
But then any test run after this will not have this setting anymore?

Reply:
Ah, you're right. I thought each test file was isolated, like a process, but that's not the case. I'll save and restore this value.

Reply:
This should be fixed now.